### PR TITLE
Ignore errno values on Windows tests

### PR DESCRIPTION
**Pull request by tvolkert**
This allows us to run some tests rather than skipping them, which
in turn allows us to focus on the more egregious test failures.

### CODE REVIEW

**Comment by tvolkert** [comment]
@goderbauer @zanderso

File: test/local_test.dart [hunk 1]
```diff
@@ -29,89 +30,48 @@ void main() {
       tmp.deleteSync(recursive: true);
     });
 
+    setUpAll(() {
+      if (io.Platform.isWindows) {
+        ignoreOsErrorCodes = true;
```
Comment (goderbauer) [inline test/local_test.dart]: Should we file an issue and add a TODO here to investigate/remove this? Not a high-prio item, but might be worth tracking.

**Comment by zanderso** [comment]
lgtm

File: test/local_test.dart (hunk 1 above)
Comment (tvolkert) [inline test/local_test.dart]: Done

**Pull request merged**